### DISCUSSION TITLE
Linux AUR nerdctl vs rancher desktop nerdctl (buildkit not working)

### DISCUSSION THREAD

**Discussion by Davincible** [rancher-sandbox/rancher-desktop#2091] [Q&A] [answered]
I just spend a good amount of time trying to figure out why my nerdctl wouldn't connect to the rancher desktop namespaces, turns out that I can't use the nerdctl installed through the AUR, and have to use the rancher destop version by selecting it in supporting utilities.

Would have been nice if this was documented, to save time. 

Why am I seeing this behavior / what is the difference in versions?

Edit: 
Also getting buildkit issues with this version;
```
$ nerdctl --namespace k8s.io build -t expressapp:v2.0 --ssh=default .

# SSH_AUTH_SOCK is actually set properly
error: invalid empty ssh agent socket: make sure SSH_AUTH_SOCK is set
FATA[0000] unrecognized image format 
```

Just found this issue (https://github.com/rancher-sandbox/rancher-desktop/issues/2072), probably buildkit related issues

**Comment by jandubois** [comment, marked as answer]
The difference is that for Rancher Desktop `nerdctl` must run inside the VM and not on the host. So the host version is just a shell wrapper to execute the command inside the VM. I'll create a doc issue from this entry...
